# HG changeset patch
# User Fabian Groffen <grobian@gentoo.org>
# Date 1522498414 -7200
#      Sat Mar 31 14:13:34 2018 +0200
# Node ID 2731ee526834ab3e907f9328c5d9ebca55e6ab95
# Parent  ded90ccbc6de298913f34d659020f1ef3b2e59cb
configure: check for tinfo matching ncurses

When we selected ncursesw, look for tinfow as well.

Bug: https://bugs.gentoo.org/651552

diff -r ded90ccbc6de -r 2731ee526834 configure.ac
--- a/configure.ac	Sat Sep 10 11:48:50 2016 +0100
+++ b/configure.ac	Sat Mar 31 14:13:34 2018 +0200
@@ -299,12 +299,14 @@ main ()
         AC_CHECK_LIB($cf_ncurses, initscr,
                 [MUTTLIBS="$MUTTLIBS -l$cf_ncurses"
 
-                AC_CHECK_LIB(tinfo, tgetent, [MUTTLIBS="$MUTTLIBS -ltinfo"])
+                if test "$cf_ncurses" = ncursesw; then
+                  AC_CHECK_LIB(tinfow, tgetent, [MUTTLIBS="$MUTTLIBS -ltinfow"])
 
-                if test "$cf_ncurses" = ncursesw; then
                   AC_CHECK_HEADERS(ncursesw/ncurses.h,[cf_cv_ncurses_header="ncursesw/ncurses.h"],
                     [AC_CHECK_HEADERS(ncurses.h,[cf_cv_ncurses_header="ncurses.h"])])
                 else
+                  AC_CHECK_LIB(tinfo, tgetent, [MUTTLIBS="$MUTTLIBS -ltinfo"])
+
                   AC_CHECK_HEADERS(ncurses/ncurses.h,[cf_cv_ncurses_header="ncurses/ncurses.h"],
                     [AC_CHECK_HEADERS(ncurses.h,[cf_cv_ncurses_header="ncurses.h"])])
 		fi],
